### PR TITLE
JBIDE-20234 - N&N for 4.3.0Beta2

### PR DESCRIPTION
**Pull request by gorkem**
JBIDE-20234- New & Noteworthy for Aerogear

### CODE REVIEW

File: documentation/whatsnew/aerogear/aerogear-news-1.2.0.Beta2.adoc [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+:page-component_id: aerogear
+:page-component_version: 1.2.0.Beta2
+:page-product_id: jbt_core
+:page-product_version: 4.3.0.Beta2
+
+== Create FeedHenry Cordova Application
+
+A new wizard allows you to convert a hybrid mobile project to a FeedHenry
+application. The wizard creates a client application on the FeedHenry cloud,
+configures the git repository for the project.
```
Comment (maxandersen) [inline documentation/whatsnew/aerogear/aerogear-news-1.2.0.Beta2.adoc]: , -> and ?

Comment (maxandersen) [inline documentation/whatsnew/aerogear/aerogear-news-1.2.0.Beta2.adoc]: got a jira ref for this change ?

Comment (maxandersen) [inline documentation/whatsnew/aerogear/aerogear-news-1.2.0.Beta2.adoc]: add line with:

```
related_jira::JBIDE-20005[]
```

**Comment by maxandersen** [comment]
fixed the , to and then pushed.

**Comment by alexeykazakov** [comment]
I don't see this PR merged in master/production. Reopening.

**Comment by maxandersen** [comment]
merged this time. not sure what happened the first time around. sorry about that.

**Pull request closed without merging**